packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -4240,13 +4246,14 @@ export class ContainerRuntime
 					contents: schemaChangeMessage,
 				};
 				this.outbox.submit({
-					contents: serializeOpContents(msg),
+					contents: this.opSerializer.stringify(msg),
```

Comment:
I would suggest adding a comment that this doesn't serialize any handles for now - that happens in the DDS. That's why it's okay to do this. It isn't obvious and might lead to thinking that we are serializing handles twice :)